docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -10,8 +10,12 @@ as those used for reporting and database development, use the JDBC driver.
 Requirements
 ------------
 
-The JDBC driver is compatible with Java versions 8 or higher, and can be used with
-applications running on Java virtual machines version 8 or higher.
+The Trino JDBC driver has the following requirements:
+
+* Java version 8 or higher. The driver can also be used with
+  applications running on Java virtual machines version 8 or higher.
```

Comment:
what does "also" indicate here? The preceding and following seem to convey same meaning.